config-freebsd.md

```diff
@@ -0,0 +1,124 @@
+  - **`rawSockets`** _(bool, OPTIONAL)_ - Allow the container to use raw sockets to support network utilities such as ping and traceroute.
+  - **`chflags`** _(bool, OPTIONAL)_ - Allow the system file flags to be changed.
+  - **`mount`** _(array of strings, OPTIONAL)_ - Allow the listed filesystem types to be mounted and unmounted in the container.
+  - **`quotas`** _(bool, OPTIONAL)_ - Allow the filesystem quotas to be changed in the container.
+  - **`socketAf`** _(bool, OPTIONAL)_ - Allow socket types other than IPv4, IPv6 and unix.
+  - **`reservedPorts`** _(bool, OPTIONAL)_ - Allow the jail to bind to ports lower than 1024.
+  - **`suser`** _(bool, OPTIONAL)_ - The value of the jail's security.bsd.suser_enabled sysctl. The super-user will be disabled automatically if its parent system has it disabled.  The super-user is enabled by default.
+
+### Mapping from jail(8) config file
+
+This table shows how to map settings from a typical `jail(8)` config file to the container configuration:
+
+| Jail parameter   | JSON equivalent      |
+| --------------   | -------------------- |
+| `jid`            | -                    |
+| `name`           | see below            |
+| `path`           | `root.path`          |
+| `ip4.addr`       | -                    |
+| `ip4.saddrsel`   | -                    |
+| `ip4`            | `freebsd.jail.ip4`   |
+| `ip6.addr`       | -                    |
+| `ip6.saddrsel`   | -                    |
+| `ip6`            | `freebsd.jail.ip6`   |
+| `vnet`           | `freebsd.jail.vnet`  |
+| `host.hostname`  | `hostname`           |
+| `host`           | `freebsd.jail.host`  |
+| `sysvmsg`        | `freebsd.jail.sysvmsg` |
+| `sysvsem`        | `freebsd.jail.sysvsem` |
+| `sysvshm`        | `freebsd.jail.sysvshm` |
+| `securelevel`    | -                    |
+| `devfs_ruleset`  | see below            |
+| `children.max`   | see below            |
+| `enforce_statfs` | `freebsd.jail.enforceStatfs` |
+| `persist`        | -                    |
+| `parent`         | `freebsd.jail.parent`  |
+| `osrelease`      | -                    |
+| `osreldate`      | -                    |
+| `allow.set_hostname` | `freebsd.jail.allow.setHostname` |
+| `allow.sysvipc`  | `freebsd.jail.allow.sysvipc` |
+| `allow.raw_sockets`  | `freebsd.jail.allow.rawSockets` |
+| `allow.chflags`  | `freebsd.jail.allow.chflags` |
+| `allow.mount`    | `freebsd.jail.allow.mount` |
+| `allow.quotas`    | `freebsd.jail.allow.quotas` |
+| `allow.read_msgbuf` | -                       |
+| `allow.socket_af` | `freebsd.jail.allow.socketAf` |
+| `allow.mlock`    | - |
+| `allow.nfsd`     | - |
+| `allow.reserved_ports` | `freebsd.jail.allow.reservedPorts` |
+| `allow.unprivileged_proc_debug` | - |
+| `allow.suser`    | `freebsd.jail.allow.suser` |
+| `allow.mount.*`  | see below            |
+
+The jail name is set to the create command's `container-id` argument.
```

Comment:
"is", SHOULD, or MUST?  (Another place that feels like documentation, not spec)

Reply:
Unless there is another mechanism to (externally from the runtime) link `jail name` with `container-id` this should be `MUST`.

Reply:
for the whole table, I'm not sure. If any of these parameters are used, the mappy is prescriptive. Is that `MUST` or `SHOULD`?